### ISSUE TITLE
Can not open classic sound options in Windows 10 (build 1903)

### ISSUE THREAD

**Issue by s3n0** [irzyxa/Volume2#63]
Hello.

I recommend using the following commands to opening the system sound settings (system control panel):
- `rundll32.exe shell32.dll,Control_RunDLL mmsys.cpl,,0`
(source info: https://winaero.com/blog/open-classic-sound-options-windows-10/)
I have verified it on Windows 10@1903, but not on Windows 7.

I don't know why, but the system sound settings (control panel) invocation from Volume2 application didn't work. I'm logged in as system administrator of course. Unfortunately it doesn't work:
- `C:\Windows\system32\mmsys.cpl`

I have also tried an alternative directory and it also doesn't work:
- `C:\Windows\Sysnative\mmsys.cpl`

BTW ... nice job this Volume2 !!!

![open the sound card settings](https://user-images.githubusercontent.com/13181404/64951646-ac515100-d87e-11e9-968d-0670419c6d28.png)

**Comment by irzyxa** [comment]
So you want to say that Playback Devices from tray menu does not work? Do you use Windows with x86 architecture?

**Comment by s3n0** [comment]
No, I use Win10 X64.

**Comment by irzyxa** [comment]
I use command:
C:\Windows\Sysnative\rundll32.exe shell32.dll,Control_RunDLL mmsys.cpl,,0

**Comment by s3n0** [comment]
Yes, this also works (since your application is 32-bit, the `C:\Windows\Sysnative` folder in win-x64 will be found). I have just tested and works. Great. Thanks!

Calling this `rundll32` also works in the application without specifying the system path `C:\Windows\Sysnative`. What strikes me is that OS ENVIRONMENT VARIABLE does not work in your application. When I specified a path using `%WINDIR%`, it didn't work. It would be better to use the path `%WINDIR%\Sysnative` because some crazy people may also have a Windows system installed on a drive other than the `C:` drive. :) But this is not such a big deal. Thanks again.
